specification/sections/04-hooks.md

```diff
@@ -6,7 +6,7 @@ toc_max_heading_level: 4
 
 # 4. Hooks
 
-[![experimental](https://img.shields.io/static/v1?label=Status&message=experimental&color=orange)](https://github.com/open-feature/spec/tree/main/specification#experimental)
+[![hardening](https://img.shields.io/static/v1?label=Status&message=hardening&color=yellow)](https://github.com/open-feature/spec/tree/main/specification#hardening)
```

Comment:
There is an open question on the order hooks that should execute. We should either address the concern in the spec or close the issue with our reasoning before changing the status of this section.

https://github.com/open-feature/spec/issues/164

Reply:
I've opened https://github.com/open-feature/spec/pull/315